soh/soh/Enhancements/randomizer/randomizer_check_tracker.cpp

```diff
@@ -867,8 +867,17 @@ void UpdateCheck(uint32_t check, RandomizerCheckTrackerData data) {
     UpdateOrdering(area);
 }
 
+void CheckTrackerWindow::Draw() {
+    if (!IsVisible()) {
+        return;
+    }
+    DrawElement();
+    // Sync up the IsVisible flag if it was changed by ImGui
+    SyncVisibilityConsoleVariable();
+}
+
 void CheckTrackerWindow::DrawElement() {
-    ImGui::SetNextWindowSize(ImVec2(400, 540), ImGuiCond_FirstUseEver);
+    ImGui::SetNextWindowSize(ImVec2(600, 375), ImGuiCond_FirstUseEver);
```

Comment:
First use window size has changed, is that intended?

And all the floating tracker/button combo stuff looks duplicated now. Copy paste error?

Reply:
Must have been. The window size was the check tracker settings window's defaults, probably copied it from the wrong spot when doing the override format. No idea how the duplication happened, though.